scenarioo-server/src/main/java/org/scenarioo/rest/builds/BuildsImporterResource.java

```diff
@@ -98,6 +95,35 @@ public void reimportBuild(@PathParam("branchName") final String branchName,
 		ScenarioDocuBuildsManager.INSTANCE.reimportBuild(buildIdentifier);
 	}
 
+	@GET
+	@Path("importBuild/{branchName}/{buildName}/{comparisonBranchName}/{comparisonBuildName}/{comparisonName}")
+	@Produces({"application/json"})
+	public Response importBuild(
```

Comment:
Changing to async is also fine for us however the importBuild method would have to return a reference ID so that the client can ask about the status for this particular import.